metabrainz/static/css/main.less

```diff
@@ -211,6 +211,26 @@ h1.page-title, h2.page-title {
   }
 }
 
+//team page 
+#team {
+  strong { color: @green; }
```

Comment:
I see no reason to change our theme there. It might be better to create a custom class like `project-name`.